tfhe/src/shortint/backward_compatibility/server_key/mod.rs

```diff
@@ -18,23 +19,13 @@ pub enum ShortintCompressedBootstrappingKeyVersions {
     V0(ShortintCompressedBootstrappingKey),
 }
 
-#[derive(Version)]
-pub struct UnsupportedCompressedServerKeyV0;
-
-impl Upgrade<CompressedServerKey> for UnsupportedCompressedServerKeyV0 {
-    type Error = crate::Error;
-
-    fn upgrade(self) -> Result<CompressedServerKey, Self::Error> {
-        Err(crate::Error::new(
-            "Unable to load CompressedServerKey, \
-            this format is unsupported by this TFHE-rs version."
-                .to_string(),
-        ))
-    }
+impl Deprecable for CompressedServerKey {
+    const TYPE_NAME: &'static str = "CompressedServerKey";
+    const MIN_SUPPORTED_APP_VERSION: &'static str = "TFHE-rs v0.8";
```

Comment:
0.9 I believe as we break things in 0.9 and some things cannot be loaded from 0.8 ?